File: Códigos em C/Conversor de Temperatura.c
#include <stdio.h>

int main()
{
    float temperatura;
    printf("Conversor de Celsius para Farenheit");
    printf("\n\nDigite a temperatura em Celsius: ");
    scanf("%f",&temperatura);
    
    float conversor = (temperatura * 1.8) + 32;
    
    printf("\nResultado da conversão: %f", conversor);
    
    return 0;
}

File: Códigos em Haskell/Conversor de Temperatura.hs
conversor x = (x * (9/5)) + 32

main = do
  putStrLn "Conversor de Temperatura"
  putStrLn "Digite uma temperatura em Celsius: "
  n1 <- readLn :: IO Float
  
  putStrLn ""
  putStrLn ("Essa temperatura em Farenheit é: " ++ show(conversor n1)++"°F")
  
